generator/urls.py

```diff
@@ -7,7 +7,7 @@
 
 urlpatterns = [
     path('', TemplateView.as_view(template_name="generator/index.html"), name='index'),
-    path('tracker/', TemplateView.as_view(template_name="tracker/tracker.html"), name='tracker'),
+    path('tracker/', views.TrackerView.as_view(), name='tracker'),
```

Comment:
I need to check if people are even using the web based tracker.  It was sort of a proof-of-concept thing I threw together and then semi-abandoned.  I've been considering dropping support for it when the beta goes live.